WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailCoordinator.swift

```diff
@@ -359,9 +359,7 @@ class ReaderDetailCoordinator {
     }
 
     private func markPostAsSeen() {
-        guard let post = post,
-              let context = post.managedObjectContext,
-              !post.isSeen else {
+        guard let post = post, !post.isSeen else {
```

Comment:
`context` was unused (as per the Xcode warning that lead me here 😅 )

I thought maybe it was important to ensure there was an `NSManagedObjectContext` attached to the object, but looking at 74c6eb35fe7ca7c638b5f4a553013ca060847e26 I think it's okay to remove it.